pcsx2/gui/ConsoleLogger.cpp

```diff
@@ -425,7 +425,7 @@ ConsoleLogFrame::ConsoleLogFrame( MainEmuFrame *parent, const wxString& title, A
 
 	menuFontSizes.AppendSeparator();
 	menuFontSizes.Append( MenuId_ColorScheme_Light,	_("&Light theme"),	_t("Default soft-tone color scheme."), wxITEM_RADIO );
-	menuFontSizes.Append( MenuId_ColorScheme_Dark,	_("&Dark theme"),	_t("Classic black color scheme for people who enjoy having text seared into their optic nerves."), wxITEM_RADIO );
+	menuFontSizes.Append( MenuId_ColorScheme_Dark,	_("&Modern Dark theme"),	_t("Modern dark color scheme which does not burn optic nerves."), wxITEM_RADIO );
```

Comment:
I don't think you need to change the menu entry text (so just `Dark theme` is probably fine).
The menu help text could be reduced to just `Modern dark color scheme.` (which someone suggested earlier I think) - It's shorter and IMO it's better to avoid humour in translatable strings since it might not necessarily translate well.

Reply:
Fair enough it was based on the Classic dark theme. I could do Dark theme (modern) for it's entry too.